controlplane/kubeadm/internal/suite_test.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package internal
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"sigs.k8s.io/cluster-api/test/helpers"
+)
+
+var (
+	testEnv *helpers.TestEnvironment
+	testCtx context.Context
+)
+
+func TestMain(m *testing.M) {
```

Comment:
I put this in `suite_test.go` to have some consistency since this is essentially bootstrapping and teardown at the package level just like `BeforeSuite` and `AfterSuite`.

Reply:
Makes sense, thanks for doing this!